client/src/locales/en.json

```
@@ -634,6 +634,91 @@
 			"retryButton": {
 				"default": "Retry connection",
 				"processing": "Connecting..."
+			},
+			"toasts": {
+				"reconnected": "Successfully reconnected to the server.",
+				"stillUnreachable": "Server is still unreachable. Please try again later."
+			}
+		}
+	},
+	"expectedValue": "Expected value",
+	"export": {
```

Biome (1.9.4)

[error] The key export was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)